TADashboard/src/main/java/base_pages/BasePage.java

```diff
@@ -0,0 +1,39 @@
+package base_pages;
+
+import com.logigear.control.common.imp.Label;
+import com.logigear.control.common.imp.Link;
+import com.logigear.driver.DriverUtils;
+
+public class BasePage {
+    private final Label lblWelcome = new Label("css=a[href='#Welcome']");
+    private final Label lblAdminister = new Label("css=a[href='#Administer']");
+    private final Link dynamicMenu = new Link("xpath=//div[@id='header']//a[text()='%s']");
+
+    public void selectWelcomeMenu(String pageName){
+        lblWelcome.click();
+        dynamicMenu.setDynamicValue(pageName);
+        dynamicMenu.click();
+    }
+
+    public void selectAdministerMenu(String pageName){
+        lblAdminister.click();
+        dynamicMenu.setDynamicValue(pageName);
+        dynamicMenu.click();
+    }
+
+    public boolean doesWelcomeLabelDisplay() {
+        return lblWelcome.isVisible();
+    }
+
+    public boolean doesAdministerLabelDisplay() {
+        return lblAdminister.isVisible();
+    }
+
+    public String getAlertText() {
```

Comment:
again, I mentioned about this many times, we don't need to wrap a method for the DriverUtils method.